internal/util/labels_annotations.go

```diff
@@ -17,14 +17,16 @@ limitations under the License.
 package util
 
 const (
-	VolumeSnapshotLabel              = "velero.io/volume-snapshot-name"
-	VolumeSnapshotHandleAnnotation   = "velero.io/csi-volumesnapshot-handle"
-	VolumeSnapshotRestoreSize        = "velero.io/vsi-volumesnapshot-restore-size"
-	CSIDriverNameAnnotation          = "velero.io/csi-driver-name"
-	CSIDeleteSnapshotSecretName      = "velero.io/csi-deletesnapshotsecret-name"
-	CSIDeleteSnapshotSecretNamespace = "velero.io/csi-deletesnapshotsecret-namespace"
-	CSIVSCDeletionPolicy             = "velero.io/csi-vsc-deletion-policy"
-	VolumeSnapshotClassSelectorLabel = "velero.io/csi-volumesnapshot-class"
+	VolumeSnapshotLabel                             = "velero.io/volume-snapshot-name"
+	VolumeSnapshotHandleAnnotation                  = "velero.io/csi-volumesnapshot-handle"
+	VolumeSnapshotRestoreSize                       = "velero.io/vsi-volumesnapshot-restore-size"
+	CSIDriverNameAnnotation                         = "velero.io/csi-driver-name"
+	CSIDeleteSnapshotSecretName                     = "velero.io/csi-deletesnapshotsecret-name"
+	CSIDeleteSnapshotSecretNamespace                = "velero.io/csi-deletesnapshotsecret-namespace"
+	CSIVSCDeletionPolicy                            = "velero.io/csi-vsc-deletion-policy"
+	VolumeSnapshotClassSelectorLabel                = "velero.io/csi-volumesnapshot-class"
+	VolumeSnapshotClassDriverBackupAnnotationPrefix = "velero.io/csi-volumesnapshot-class"
```

Comment:
Looks like `VolumeSnapshotClassDriverBackupAnnotationPrefix` and `VolumeSnapshotClassDriverPVCAnnotation` are duplicated with `VolumeSnapshotClassSelectorLabel`.

Reply:
Actually all 3 variables including VolumeSnapshotClassSelectorLabel  have the same string
I kept that intentionally same to make user experience easier
While on the other hand - all are separate 3 variables such that it is easy to change in code than reusing the variables.

Do you have any suggestion for keeping the values different?